### DISCUSSION TITLE
Update of VM running status

### DISCUSSION THREAD

**Discussion by SviridoffA** [apache/cloudstack#9541] [Ideas]
Hello everyone! I need help, and will be very grateful if someone can tell me if there is any way to speed up updating the status of a virtual machine turned off from the guest system? As far as I understand, this status is determined through out of band monitoring, and updating the status takes about three minutes, which causes a little confusion. This process can be accelerated by repeatedly pressing the shutdown of the virtual machine from the graphical interface, but this is not obvious to the simple user and not very convenient. Perhaps there is some setting that can affect this?

I am using:
Cloudstack 4.19.1.1
Hypervisor KVM

**Comment by rajujith** [comment]
@SviridoffA  can you try reducing the 'sync.interval' in global settings and see if it improves?

**Reply by SviridoffA** [reply]
Thanks for your reply. I have tryed to change this setting? It was 60 seconds, I have set to 10. After this i have restarted managment servers, but seems nothing changed. Status updating in 2 - 3 minutes anyway.

**Comment by DaanHoogland** [comment]
@SviridoffA , did you find a workaround for your issue? (the power reports are this way by design, every 1 minute with three retries)

**Reply by SviridoffA** [reply]
Unfortunately, I don’t have a solution of this problem at the moment. Most users are accustomed to duplicating the shutdown process from the UI in order to speed things up. This is a bit confusing me because I came to CloudStack about a year ago from the VMware world, where there weren’t similar issues. It would be great if there was some way to update the status, perhaps a supplementary button or just by clicking a “Refresh” button when we managing VM from UI. Anyway, since this behavior is by design and it seems there’s no solution, I think we can close this discussion.

**Reply by DaanHoogland** [reply]
That sounds like a vlid feature request. I do not have a clear vision on implementation though. Let’ s keep this open and mark it as an idea for future development.
